torch/optim/lr_scheduler.py

```diff
@@ -457,50 +510,38 @@ class WarmUpLR(_LRScheduler):
         >>> # lr = 0.0375   if epoch == 2
         >>> # lr = 0.04375  if epoch == 3
         >>> # lr = 0.005    if epoch >= 4
-        >>> scheduler = WarmUpLR(self.opt, warmup_factor=0.5, warmup_iters=4, warmup_method="linear")
+        >>> scheduler = LinearLR(self.opt, start_value=0.5, total_iters=4)
```

Comment:
I agree leaving the end param out of the example. It's very rare that end != 1.0 and can lead to strange behaviours.